<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssemblerFiles</class>
 <widget class="QDialog" name="AssemblerFiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="assemblyCodeFileLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Assembly Code File</string>
   </property>
  </widget>
  <widget class="QLabel" name="textDumpFileLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Text Dump File</string>
   </property>
  </widget>
  <widget class="QLabel" name="memoryDumpFileLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Memory Dump File</string>
   </property>
  </widget>
  <widget class="QLabel" name="registerUpdateFileLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Register Update File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="assemblyCodeFile">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textDumpFile">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextDump.bin</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="memoryDumpFile">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MemoryDump.bin</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="registerUpdateFile">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>RegisterUpdate.txt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Assemble</string>
   </property>
  </widget>
  <widget class="QLabel" name="textDumpFileSetToDefault">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(set to default)</string>
   </property>
  </widget>
  <widget class="QLabel" name="memoryDumpFileSetToDefault">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(set to default)</string>
   </property>
  </widget>
  <widget class="QLabel" name="registerUpdateFileSetToDefault">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(set to default)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
